multi_imbalance/resampling/GlobalCS.py

```diff
@@ -0,0 +1,60 @@
+from collections import Counter, defaultdict
+
+import numpy as np
+import sklearn
+from sklearn.neighbors import NearestNeighbors
+
+
+class GlobalCS(object):
+    """
+    Global CS is an algorithm that equalizes number of samples in each class. It duplicates all samples equally
+    for each class to achieve majority class size
+    """
+
+    def __init__(self) -> None:
```

Comment:
Do we need this type hint? __init__ by definition should not return anything, so I thinks it's useless here